src/main/java/no/fdk/rabbit/RabbitMQListener.java

```diff
@@ -0,0 +1,28 @@
+package no.fdk.rabbit;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.fdk.configuration.FusekiConfiguration;
+import no.fdk.model.fuseki.action.CompactAction;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class RabbitMQListener {
+    private final CompactAction compactAction;
+    private final FusekiConfiguration fusekiConfiguration;
+
+    @RabbitListener(queues = "#{queue.name}")
+    public void receiveConceptPublisher(Message message) {
+        String routingKey = message.getMessageProperties().getReceivedRoutingKey();
+        log.info(String.format("Received message from key: %s", routingKey));
+        String storePath = fusekiConfiguration.getStorePath();
+        fusekiConfiguration.getDatasetNames()
+                .stream()
+                .map(dataset -> "%s/%s".formatted(storePath, dataset))
+                .forEach(compactAction::doCompactForDatasetPath);
+    }
+}
```

Comment:
Kjør gjerne på med litt luft

```suggestion
    public void receiveConceptPublisher(Message message) {
        String routingKey = message.getMessageProperties().getReceivedRoutingKey();

        log.info(String.format("Received message from key: %s", routingKey));

        fusekiConfiguration
                .getDatasetNames()
                .stream()
                .map(dataset -> "%s/%s".formatted(fusekiConfiguration.getStorePath(), dataset))
                .forEach(compactAction::doCompactForDatasetPath);
    }
}

```